docs-site/content/0.19.0/guide/features/scoped-API-keys.md

```diff
@@ -0,0 +1,47 @@
+# Scoped API Keys
+
+API keys are at the core of Typesense. To perform any action with Typesense, you need API keys. Typesense also allows access control on API keys. You can define capabilities as to what a user can or cannot do. You can also restrict access to a specific document or collection. In the case of a multi-tenant environment, you can scope API keys to a particular subset. This is helpful when you have indexed data from multiple tenants in your Typesense server and want to restrict users to only access their subset of data.
```

Comment:
Instead of :

> API keys are at the core of Typesense.

Typesense is designed with security and fine-grained access control in mind.